README.md

```diff
@@ -1,2 +1,52 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
 # ci-public
+
 CI/CD related files
+
+## Directories
+
+| Directory | Description
+| --- | ---
+| addtimestamp | Tool for adding timestamp to build report .json
+| configs | Some Jenkins configs?
+| containers | Everything related to dockerized server solution
```

Comment:
Please confirm, and remove question mark before merging.

Reply:
Ah, forgot about those already. Will do.